dashboards-observability/public/components/event_analytics/explorer/visualizations/workspace_panel/workspace_panel.scss

```diff
@@ -20,6 +20,6 @@
 }
 
 #vis__mainContent .ws__central--canvas {
-  background-color: #FFF;
-  border-right: 1px solid #D3DAE6;
-}
\ No newline at end of file
+  background-color: #fff;
```

Comment:
Is there any difference in lower and upper cases?

Reply:
CSS values are case insensitive. Thus, it is not a good practice to go with Uppercase, so going with the norm.